docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst

```diff
@@ -75,14 +75,38 @@ Start each Volttron instance after configuration. The "-l" option in the
 following command tells volttron to log to a file. The file name
 should be different for each instance.
 
-::
+.. code-block:: console
 
     $ volttron -l log1&
 
+VOLTTRON Central need to accept the connecting instances'
```

Comment:
VC auto adds the /.*/ on startup...the other side where vcp is on an external platform does not.